app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java

```diff
@@ -70,6 +73,13 @@ public void onBindViewHolder(@NonNull PictureViewHolder holder, int position) {
     }
 
     private void handleLike(@NonNull PictureViewHolder holder,Post post){
+        if (post.getUid().equals(Profile.getActiveProfile().getUid())) {
+            holder.like.setVisibility(View.GONE);
+            return;
+        }
+
+        holder.like.setVisibility(View.VISIBLE);
+
```

Comment:
I don't think needed to hide the like button on your own publications. In any other social network you can like your posts

Reply:
I think that it is better for the display and the UI logic to be placed at the same position, so to not put too many buttons on the bottom I removed the like for the user's profile.

Moreover, to me it makes no sense for a user to like a picture, but we can ask the others what they think about it.